test/unstable/listParSafe.chpl

```diff
@@ -0,0 +1,12 @@
+use List;
+
+// show that the 'parSafe' argument triggers an unstable warning on all 'list' initializers:
```

Comment:
Should the code below also include a use of the initializer `init(other:list)` ?

Reply:
It does on line 5.